ft/txn/rollback.cc

```diff
@@ -92,7 +92,7 @@ static inline PAIR_ATTR make_rollback_pair_attr(long size) {
 PAIR_ATTR
 rollback_memory_size(ROLLBACK_LOG_NODE log) {
     size_t size = sizeof(*log);
-    if (&log->rollentry_arena) {
+    if (1) { // if (&log->rollentry_arena) {
```

Comment:
That line should be deleted.
On Aug 24, 2015 1:10 PM, "Christian" <notifications@github.com> wrote:

> In ft/txn/rollback.cc
> <https://github.com/percona/PerconaFT/pull/310#discussion_r37775191>:
>
> > @@ -92,7 +92,7 @@ static inline PAIR_ATTR make_rollback_pair_attr(long size) {
> >  PAIR_ATTR
> >  rollback_memory_size(ROLLBACK_LOG_NODE log) {
> >      size_t size = sizeof(*log);
> > -    if (&log->rollentry_arena) {
> > +    if (1) { // if (&log->rollentry_arena) {
>
> Do we want to remove this predicate check? I assume the rollentry_arena is
> always non-null and the predicate check can be removed completely.
>
> ...Or maybe this change is related to some other work and snuck in by
> accident?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/percona/PerconaFT/pull/310/files#r37775191>.
>